_sass/search.scss

```diff
@@ -363,4 +363,141 @@ input::placeholder {
     }
   }
 }
+
+.hubSearchWrapper {
+  @include desktop {
+    left: 0;
+    top: 8px;
+    margin-left: 0;
+    position: absolute;
+  }
+  @media screen and (max-width: 474px) {
+    .algolia-autocomplete .ds-dropdown-menu {
+      min-width: 300px;
+   }
+  }
+}
+
+#hub-search-icon {
+  background-image: url(/assets/images/search-icon.svg);
+  color: transparent;
+  opacity: 0.4;
+  width: 25px;
+  height: 25px;
+  background-size: 12px 20px;
+  background-repeat: no-repeat;
+  background-position: 7px 5px;
+  border-radius: 25px;
+  margin-left: 20px;
+  cursor: pointer;
+  &:hover {
+    background-image: url(/assets/images/search-icon-orange.svg);
+    opacity: 1;
+  }
+  @include desktop {
+    margin-left: 50px;
+    background-position: 7px 4px; 
+  }
+  @include full-nav-menu-desktop {
+    margin-left: 50px;
+  }
+  @media screen and (max-width: 767px) {
+    position: absolute;
+    left: 20px;
+    top: 1px;
+  }
+}
+
+input#hub-search-input {
+  background-color: inherit;
+  border: none;
+  color: $black;
+  font-size: rem(18px);
+  font-weight: 300;
+  line-height: 20px;
+  outline: none;
+  padding-left: 25px;
+  position: relative;
+  display: none;
+  width: 550px;
+  @include small-desktop {
+    padding-left: 0;
+  }
+  @include desktop {
+    padding-left: 0;
+    width: 450px;
+  }
+  @media screen and (max-width: 767px) {
+    margin-left: 0;
+    padding-left: 5px;
+    width: 490px;
+  }
+  @media screen and (min-width: 475px) and (max-width: 579px) {
+    width: 390px;
+  }
+  @media screen and (max-width:474px) {
+    width: 290px;
+  }
+}
+
+#hub-close-search {
+  display: none;
+  margin-left: 20px;
+  opacity: 0.4;
+  cursor: pointer;
+  &:hover {
+    color: $orange;
+    opacity: 1;
+  }
+  @include desktop {
+    margin-left: 35px;
+  }
+  @media screen and (max-width: 767px) {
+    margin-left: 0;
+  }
+}
+  
+.hub-input {
+  background-color: red;
+  padding-left: 0;
+  width: 500px;
+}
+
+.active-hub-divider{
+  border-color: $orange;
+  @include desktop {
+    margin-top: 2.5rem;
+    margin-left: 0;
+    width: 600px;
+  }
+  @include desktop {
```

Comment:
You can probably remove one of the widths that are inside of these desktop mixins.